src/Bounds.cpp

```diff
@@ -1276,6 +1288,18 @@ class Bounds : public IRVisitor {
         } else if (op->is_intrinsic(Call::return_second)) {
             internal_assert(op->args.size() == 2);
             interval = arg_bounds.get(1);
+        } else if (op->is_intrinsic(Call::strict_float)) {
+            internal_assert(op->args.size() == 1);
+            interval = arg_bounds.get(0);
+        } else if (op->is_intrinsic(Call::round)) {
```

Comment:
Round was already handled below more tightly than this (you can just round the min and max). strict_float should probably be handled in the same place. i.e. we should probably preserve the strict_float wrapper around the min and max so that floating point optimizations don't happen to the min and max in a way that makes them no longer contain the original expression.

Reply:
Ah, I missed the `Call::round` handling below -- I think the case I saw didn't pass the `(interval = arg_bounds.get(0)).is_bounded()` test and that's why I thought it was missing.